packages/components/src/core/Dropdown/index.tsx

```diff
@@ -28,93 +34,101 @@ export type Value<T, Multiple> = Multiple extends undefined | false
   ? T | null
   : Array<T> | null;
 
-export type MUIValue<Multiple> = AutocompleteValue<
-  DefaultDropdownMenuOption,
-  Multiple,
-  undefined,
-  undefined
->;
+// (masoudmanson): This can be removed since we no longer need it
+export type MUIValue<
+  T extends DefaultAutocompleteOption,
+  Multiple extends boolean | undefined,
+  DisableClearable extends boolean | undefined,
+  FreeSolo extends boolean | undefined
+> = AutocompleteValue<T, Multiple, DisableClearable, FreeSolo>;
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-type RenderFunctionType = (props: any) => JSX.Element;
-
-export interface DropdownProps<Multiple extends boolean | undefined> {
+// type RenderFunctionType = (props: any) => JSX.Element;
```

Comment:
dead code?

Reply:
Oh no, it seems I've become quite sloppy lately 😥

Reply:
Nah with a big PR like this, it's normal to miss some stuff 😄 👍  That's why we have code reviews and look out for each other 💪 💪